conans/client/source.py

```diff
@@ -9,7 +9,21 @@
 from conans.model.conan_file import get_env_context_manager
 from conans.model.scm import SCM, get_scm_data
 from conans.paths import CONANFILE, CONAN_MANIFEST, EXPORT_SOURCES_TGZ_NAME, EXPORT_TGZ_NAME
-from conans.util.files import (clean_dirty, is_dirty, load, mkdir, rmdir, set_dirty, walk)
+from conans.util.files import (clean_dirty, is_dirty, load, mkdir, rmdir, set_dirty,
+                               merge_directories)
+
+
+class PackageLocalLayout(object):
+    ref = None
```

Comment:
seems like not used anywhere?

Reply:
Yes, it is used [here](https://github.com/conan-io/conan/pull/4901/files#diff-802aae676800bf9b0e36c4909618951dR117)